$NetBSD$

cmake can't find the file it's looking for so just ignore it

--- CMakeLists.txt.orig	2020-05-07 12:44:38.000000000 +0000
+++ CMakeLists.txt
@@ -66,7 +66,7 @@ if(EXISTS "${PROJECT_SOURCE_DIR}/.git")
 	string(REGEX REPLACE "-([0-9]*).*" ".\\1" VERSION "${GIT_VERSION}")
 	file(WRITE ${SOURCE_VERSION_CACHE} "${VERSION}")
 else()
-	file(STRINGS ${SOURCE_VERSION_CACHE} VERSION)
+#	file(STRINGS ${SOURCE_VERSION_CACHE} VERSION)
 endif()
 
 # configure project name and version
